hypervisor/boot/guest/vboot_wrapper.c

```diff
@@ -61,15 +61,16 @@ void init_vboot_operations(void)
 			}
 		}
 	}
-}
 
-/* @pre: vboot_ops->init != NULL */
-void init_vboot(void)
-{
+	/*
+	 * vboot_ops is mandatory and it will be initialized correctly.
+	 * The @pre vboot_ops->init is called to assure that the boot env is
```

Comment:
What does @pre mean here?

It should be before the beginning of the API.

Reply:
The @pre will be removed.
Sorry that I just copied it from another place to mention that it should be executed firstly.
It seems that it is not used correctly.